### ISSUE TITLE
Unable to permanently delete account

### ISSUE THREAD

**Issue by micmalti** [ndom91/briefkasten#64]
I appreciate the work that you've been doing on this project, but there are some features that still need to be ironed out before switching from raindrop. For the time being, I wish to delete my account, yet I couldn't find the option in the account settings.

**Comment by ndom91** [comment]
Thanks for giving it a shot! What were some of the things that you found missing?

Regarding deleting your account, feel free to email me at briefkasten@ndo.dev :+1:

**Comment by micmalti** [comment]
Sent email. Thanks for reaching out. As to the issues that I've encountered: 

1. There doesn't seem to be an option to replace the cover images with custom ones (from local storage or a given URL).
2. On Firefox version 121.0, I couldn't get the extension submitted by [rauenzi](https://github.com/rauenzi) to link up to the public instance. For me, it's also important that the extension is available for the mobile version of Firefox in order to have access to my bookmarks on my phone.
3. I ran into problems with deleting bookmarks, though I'm not sure if this issue is solely present in the public instance of the app.

I can understand if you consider these features not to be a priority at the moment, but hopefully they can be implemented at a later stage. Wishing you all the best with this project.
